app/lib/server/startup/settings.ts

```diff
@@ -3219,8 +3219,8 @@ settingsRegistry.addGroup('VoIP', function () {
 				},
 			});
 
-			this.add('VoIP_Management_Server_Password', 'secure_password', {
-				type: 'string',
+			this.add('VoIP_Management_Server_Password', '', {
```

Comment:
These values were left as references of what should be on those settings 🤔 any reason to remove them?

Reply:
Since they were actual values instead of placeholders, we don't usually use values like this

Reply:
Fair enough, can we add some indication for the format of hostnames as placeholder text? So they know where to put a domain url (a.b.c) or a websocket url 👀